### PR TITLE
is work

### PR DESCRIPTION
**Pull request by x183**
yes,
much good :100:

### CODE REVIEW

**Review by Supergamer1337 — APPROVED** [review]
I actually hate this but OK

File: backend/src/services/expressService.ts [hunk 1]
```diff
@@ -6,14 +6,19 @@ import playRouter from '../routers/playRouter.js';
...
+import { getPlayStatus } from './playService.js';
+import { emitEvent, initWebsocket } from './websocketService.js';
 
 export const createServer = async () => {
 	const app = express();
 	app.use(cors());
 	setupGlobalMiddleware(app);
 	setupRoutes(app);
 	const server = initWebsocket(app);
+	setInterval(async () => {
```
Comment (x183) [inline backend/src/services/expressService.ts:18]: I know this pr is already merged, but this introduces a bug, where the backend tries to call spotify before updating our token, leading to it crashing within 1 second of startup.

Comment (Supergamer1337) [reply backend/src/services/expressService.ts:18]: ...

Comment (Supergamer1337) [reply backend/src/services/expressService.ts:18]: I tried it but that didn't happen? Maybe it was reusing the token from before?

Comment (Supergamer1337) [reply backend/src/services/expressService.ts:18]: But yes, this should probably be fixed

Comment (x183) [reply backend/src/services/expressService.ts:18]: A very simple fix is to just check the auth token before each access, but that's not a great solution...

**Pull request merged**